lib/hyrax/configuration.rb

```diff
@@ -379,12 +379,28 @@ def extract_full_text?
       @extract_full_text = true
     end
 
+    attr_writer :uploader
+    def uploader
+      return default_uploader_config unless Rails.env.development?
+      # use sequential uploads in development to avoid database locking problems with sqlite3.
+      default_uploader_config.merge!(limitConcurrentUploads: 1, sequentialUploads: true)
```

Comment:
The implemention I am moving uses `merge!`. I don't need to use it, but I wasn't planning to change it. I can do that in my next push.